src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -71,20 +73,32 @@ const getParcelsForDelivery = async (parcelIds: string[]): Promise<ParcelsForDel
             return { data: null, error: { type: "noMatchingClient", logId: logId } };
         }
 
+        if (parcel.collection_centre === null) {
+            const logId = await logErrorReturnLogId(
+                "Error with fetch: Parcels. No collection centre found"
+            );
+            return { data: null, error: { type: "noCollectionCentre", logId: logId } };
+        }
+
```

Comment:
I added in this error for if no collection centre was found (which includes delivery), but I don't know how to test / QA this as there isn't a way of deleting a collection centre?

Reply:
How does this behave with deliveries? If those don't have a collection centre I think we shouldn't throw an error (but we should if it's isDelivery==false and collectionCentre==null)

In terms of testing - QA may be tricky but I think you can manually delete the value of the field in the database through supabase UI (i.e. not the actual collection centre but just the value for some specific record just make it null)